app/views/layouts/_footer.html.erb

```diff
@@ -0,0 +1,15 @@
+<footer class="footer">
+  <small>
+    <%= I18n.t("the") %>
```

Comment:
- t "the"
- Không cần dùng I18n. ở trước trừ khi ở trong js nhé sửa tương tự cho các chỗ còn lại